packages/react-menu/src/components/Menu/useMenuContextValues.test.tsx

```diff
@@ -0,0 +1,17 @@
+import { renderHook } from '@testing-library/react-hooks';
+import * as React from 'react';
+
+import { useMenu } from './useMenu';
+import { useMenuContextValues } from './useMenuContextValues';
+
+describe('useAccordionContextValues', () => {
+  it('should return a value for "menu"', () => {
+    const { result } = renderHook(() => {
+      const state = useMenu({ children: <span /> });
+
+      return useMenuContextValues(state);
+    });
+
+    expect(result.current.menu).toBeDefined();
```

Comment:
I think an inline snapshot could work here, and wouldn't be too big

Reply:
Added 👍